Slim/Player/Protocols/HTTPS.pm

```diff
@@ -85,6 +85,12 @@ sub canDirectStreamSong {
 	return 0;
 }
 
+sub slimprotoFlags {
+	my ($client, $url, $isDirect) = @_;
```

Comment:
Would that still work if the URL actually was http (remember TIDAL?...)?

Reply:
you're absolutely right, I forgot my own doing :-)